spec/factories/users.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :user do
+    name { "Name" }
+    surname { "Surname" }
+    email { "email@example.com" }
+    password { "123456" }
+
+    trait :role_admin do
+      role { "admin" }
+    end
+
+    trait :role_user do
+      role { "user" }
+    end
+  end
+end
```

Comment:
You can use role user by default

```suggestion
FactoryBot.define do
  factory :user do
    name { "Name" }
    surname { "Surname" }
    email { "email@example.com" }
    password { "123456" }
    role { "user" }

    trait :admin do
      role { "admin" }
    end
  end
end
```